### ISSUE TITLE
Website Design

### ISSUE THREAD

**Issue by RichardLitt** [ipld/website#7]
It would be good to have a discussion, or a call, to establish website goals and constraints, and to establish the website design, in writing.

I would specifically like to have: a page tree, a design trajectroy, and to go over any current skeletons. 

@dignifiedquire Are you free next week to go over this with me? Perhaps we should add it to the agenda for Monday?

**Comment by dignifiedquire** [comment]
@RichardLitt lets talk about this in the next days 👍

**Comment by RichardLitt** [comment]
Some notes:

# IPLD

Participants:
    @RichardLitt
    @dignifiedquire

https://github.com/protocol/team-week-2016-Q3/issues/33
It would be good to have a discussion, or a call, to establish website goals and constraints, and to establish the website design, in writing.
I would specifically like to have: a page tree, a design trajectroy, and to go over any current skeletons.

Website already exists; http://ipld.io/
- Doesn't update automatically
- Single page 

### Page Tree
- Landing page: 
- short introduction
- list of implementations
- Playground (waiting on @jbenet)
- Spec (can do today)

### To Do
- [ ] Remake in Hugo
- [ ] Make a spec page
- [ ] Make a list of implementations

https://projects.invisionapp.com/share/WX7N31QK4#/screens/144831191

**Comment by victorb** [comment]
@RichardLitt @dignifiedquire @nicola 

I've done a quick prototype with some more content and with the existing IPLD logo, just to experiment. This would be a lightweight version we can go live with, while waiting for a proper design. What you think? (imagine in your mind that it doesn't say "Implementations" in the header and also Examples would be "Install / Examples")

![](https://ipfs.io/ipfs/QmUEvM3GNNnNsh24MFBiax29euhuMGtjQMurfaEvSfoGzY)

**Comment by RichardLitt** [comment]
@VictorBjelkholm I think this looks good. Some questions:
- What would the languages link to?
- What about the list of all of the IPLD repositories? 
- What about a contribute section? What will people _do_ with this information, and how do we facilitate that?

**Comment by dignifiedquire** [comment]
Some quick thoughts
- I don't think we should add any kind of logo, we have a pipeline for getting a proper logo for IPLD in this quarter, so using one and then another would be confusing. Lets stay with just type for now, it is completely fine.
- I don't think the color you chose works. It's too bright and the logos and icons are majorly distracting while trying to read the text
- Why did you change the font from my draft?

**Comment by victorb** [comment]
Thanks a lot for the feedback!

>    What would the languages link to?

To the implementation in that language. 

>    What about the list of all of the IPLD repositories?

On the website we probably don't need to have a list of all the repositories right now. We could instead just have the link to Github, then have the list there.

>    What about a contribute section? What will people do with this information, and how do we facilitate that?

Contribute & Community section would basically just be a section information people that IPLD is open source and if you go _here_ you can find out more.

>    I don't think we should add any kind of logo, we have a pipeline for getting a proper logo for IPLD in this quarter, so using one and then another would be confusing. Lets stay with just type for now, it is completely fine.
>    I don't think the color you chose works. It's too bright and the logos and icons are majorly distracting while trying to read the text

Makes perfect sense and I agree. I'll address this.

>    Why did you change the font from my draft?

It was a quick prototype to start with, if you feel strongly about it, send me the font and I'll switch it.

**Comment by victorb** [comment]
Closing this in favor of the specific issues we have now.
